scripts/start.js

```diff
@@ -48,25 +48,28 @@ let isElectronRunning = false;
  * Prevents multiple re-runs of Electron App
  */
 function runElectronApp() {
-  if (isElectronRunning)
-    return;
+  if (isElectronRunning) return;
 
   isElectronRunning = true;
+  // For Windows Support
+  // Here cross-env can be used but
+  // since its not that big of a control I checked manually
+  // Maybe some env variable like `isWin` can be passed to the procces for easy use???
+  const command = /^win/.test(process.platform)
+    ? `set ELECTRON_START_URL=http://localhost:${DEFAULT_PORT} && electron .`
+    : `ELECTRON_START_URL=http://localhost:${DEFAULT_PORT} electron .`;
```

Comment:
This ternary could be avoided by passing the environment variable to `exec` (I believe you can pass an option object which can have an `env` key.

Reply:
> This ternary could be avoided by passing the environment variable to exec (I believe you can pass an option object which can have an env key.

So there is indeed an option to specify `env`, but there appears to be a bug with Electron where not only are the values not passed, but it messes a bunch of stuff up (commands like `ls` or `pwd` fail). Discussion [here](https://github.com/electron/electron/issues/3627).

At least, that was my experience with `spawn`. Given that, though, I'm happy to just have a ternary here :)

Reply:
@joshwcomeau Added a bounty on your SO question. Maybe we can get an answer for this issue there since the Github issue doesn't have any traction.